tests/test_integration.py

```diff
@@ -96,7 +97,7 @@ def cb_poisson_regressor_model():
 def test_poisson_regression():
     np.random.seed(42)
 
-    df = pd.read_csv("./tests/integration_test_data.csv")
```

Comment:
Was the path specified like this for a reason? Locally my tests were failing so that's why I adjusted the path. Let me know if you want me to change this back.

Reply:
It is a relative path, so it depends on your working directory from which you call the tests. As we call it from the repository root, we need the ./tests/ before the csv file. So, please change back.